kolibri/tasks/api.py

```diff
@@ -190,6 +183,18 @@ def localdrive(self, request):
 
         return Response(out)
 
+    @list_route(methods=['get'])
+    def channelinfo(self, request):
```

Comment:
any opinions if this endpoint should go somewhere else besides `TasksViewSet`? @rtibbles

Reply:
As it launches a task, it seems ok - I would just be resistant to adding anything that isn't explicity about tasks to this ViewSet.

The other question about the `TasksViewSet` is when does it stop becoming practicable to just keep adding list routes to the same viewset for every backend task we want to allow?

Reply:
I would say anything that isn't a long running task, that wants to communicate with studio, then maybe it would be wise to move that functionality elsewhere.

Reply:
@rtibbles the original design of the tasks api is to allow anyone to trigger *any* management command by just making an API request -- but the more I think about it, the more that seems to be a big vulnerability for Kolibri. So I do lean on creating a new endpoint for every task we want to potentially schedule through the frontend. It's cumbersome, but I also don't foresee adding too many too often.

Reply:
Yeah, I don't think letting the API trigger arbitrary management commands is desirable either :)

My question would be whether we have some other way of enumerating possible tasks that does not require a new list route.

Also, whether this is something that other plugins could do - register a task that can be executed via the API.

Reply:
There might be a way to generalize the tasks enumeration endpoint systematically -- one thing that celery does, is to add a decorator for functions that can be called asynchronously. That's one possible approach that I like. But we can design that outside of the scope of this PR.

Reply:
As for supporting tasks from plugins, that's a great idea! Adding decorators also work, or possibly calling a `register_task` function on the Python side, that automatically registers that function as an API endpoint.